src/libs/ui/settingsdialog.cpp

```diff
@@ -330,5 +334,12 @@ void SettingsDialog::saveSettings()
     settings->proxyUserName = ui->proxyUsernameEdit->text();
     settings->proxyPassword = ui->proxyPasswordEdit->text();
 
+    if (settings->isIgnoreSSLErrorsEnabled != ui->ignoreSSLErrorsCheckBox->isChecked()) {
+        // https://forum.qt.io/topic/125873/how-to-clear-qnetworkreply-ignoresslerrors/2
+        m_application->networkManager()->clearAccessCache();
+    }
```

Comment:
This is already done in [`Core::Application::applySettings`](https://github.com/zealdocs/zeal/blob/94247fd4d03e6ae079cae8e50e6eb9a67fc0afad/src/libs/core/application.cpp#L274-L275), but needs to be moved outside of proxy selection switch.

Reply:
Done - https://github.com/zealdocs/zeal/pull/1521/commits/c3be7f451322005604ec6b174e5fdb413793981b .